test/output-app/model/Tweet.js

```diff
@@ -1,60 +1,158 @@
-import DataLoader from 'dataloader';
-import findByIds from 'mongo-find-by-ids';
+/* eslint-disable prettier */
+import {
+  queryForRoles,
+  onAuthRegisterLoader,
+  authlog,
+  checkAuthDoc
+} from 'create-graphql-server-authorization';
 
 export default class Tweet {
   constructor(context) {
     this.context = context;
     this.collection = context.db.collection('tweet');
     this.pubsub = context.pubsub;
-    this.loader = new DataLoader(ids => findByIds(this.collection, ids));
+    const { me, User } = context;
+    queryForRoles(
+      me,
+      ['admin', 'world'],
+      ['authorId', 'coauthorsIds'],
+      { User },
+      onAuthRegisterLoader('tweet findOneById', 'readOne', me, this)
+    );
   }
 
-  findOneById(id) {
-    return this.loader.load(id);
+  async findOneById(id, me, resolver) {
+    const log = authlog(resolver, 'readOne', me);
+    if (!this.authorizedLoader) {
```

Comment:
I don't understand why it would ever be allowed that you do not have an `authorizedLoader`.